packages/ui/src/helpers/authenticator/facade.ts

```diff
@@ -100,11 +101,24 @@ export const getServiceContextFacade = (state: AuthMachineState) => {
     }
   })();
 
+  const authStatus = ((route) => {
+    switch (true) {
```

Comment:
Rather than using `true` as the argument here, can this be `route`?

Reply:
Updated

Reply:
ad0eb12cdf3de26d9143b8dd5b2176079db3d237